vscode-dotnet-runtime-extension/CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Performance improvements.
 
 Fixes a bug with apt-get lock holding (status 100) on Linux with package management.
 
+Adds acquireStatus support for ASP.NET runtime installs.
+
+Other bug fixes.
```

Comment:
nit: Maybe `Minor bug fixes.` would be a bit more descriptive?

Reply:
I think that is a better way of putting it. :)

Reply:
It got auto-merged though so I will fix it next time 🤦